### PR TITLE
(IMAGES-201) Disable automatic apt updates on debuntus

### PR DESCRIPTION
**Pull request by shrug**

### CODE REVIEW

**Comment by melissa** [comment]
Looks right! Is there a way we can test this before merging it?

File: manifests/modules/packer/files/vsphere/periodic [hunk 1]
```diff
@@ -0,0 +1,3 @@
+#  APT::Periodic::Enable "1";
+#  - Enable the update/upgrade script (0=disable)
+APT::Periodic::Enable "0";
```
Comment (underscorgan) [inline manifests/modules/packer/files/vsphere/periodic]: missing newline

File: manifests/modules/packer/manifests/vsphere/repos.pp [hunk 2]
```diff
@@ -4,6 +4,13 @@
 
     debian: {
 
+      file { $periodic_file:
+        owner   => 'root',
+        group   => 'root',
+        mode    => '0644',
+        source  => 'puppet:///modules/packer/vsphere/periodic',
+      }
```
Comment (underscorgan) [inline manifests/modules/packer/manifests/vsphere/repos.pp]: Does this file need to have any dependencies or can we sorta lay it down whenever?

Comment (shrug) [inline manifests/modules/packer/manifests/vsphere/repos.pp]: This impacts a system default script that runs in cron.daily, so it really has no other deps at build time

File: manifests/modules/packer/files/vsphere/periodic (hunk 1 above)
Comment (shrug) [inline manifests/modules/packer/files/vsphere/periodic]: git may not be displaying those, or stripping them - it exists in my local file

File: manifests/modules/packer/manifests/vsphere/repos.pp (hunk 2 above)
Comment (underscorgan) [inline manifests/modules/packer/manifests/vsphere/repos.pp]: 👍

File: manifests/modules/packer/files/vsphere/periodic (hunk 1 above)
Comment (underscorgan) [inline manifests/modules/packer/files/vsphere/periodic]: hm, it's showing up now?

**Comment by underscorgan** [comment]
@shrug is this ready for merge?

**Comment by shrug** [comment]
I think so, yes

**Pull request merged**